app/org/sagebionetworks/bridge/models/ForwardCursorPagedResourceList.java

```diff
@@ -1,88 +1,75 @@
+ * A page of items from a longer list of items, as calculated based on the supplied 
+ * request parameters, with a <code>nextPageOffsetKey</code> to retrieve the next page 
+ * of records. This kind of list cannot calculate the total number of records across 
+ * all pages, or provide the information to page backwards in the list.
  */
-public class ForwardCursorPagedResourceList<T> {
+public class ForwardCursorPagedResourceList<T> extends ResourceList<T> {
 
-    private final List<T> items;
-    private final int pageSize;
-    private final @Nullable String offsetKey;
-    private final Map<String,String> filters = Maps.newHashMap();
+    private static final String HAS_NEXT = "hasNext";
+    
+    private final @Nullable String nextPageOffsetKey;
 
     @JsonCreator
     public ForwardCursorPagedResourceList(
-            @JsonProperty("items") List<T> items, 
-            @JsonProperty("offsetKey") String offsetKey,
-            @JsonProperty("pageSize") int pageSize) {
-        this.items = items;
-        this.offsetKey = offsetKey;
-        this.pageSize = pageSize;
+            @JsonProperty(ITEMS) List<T> items, 
+            @JsonProperty(NEXT_PAGE_OFFSET_KEY) String nextPageOffsetKey) {
+        super(items);
+        this.nextPageOffsetKey = nextPageOffsetKey;
     }
-
-    /**
-     * A convenience method for adding filters without having to construct an intermediate map of filters.
-     * e.g. PagedResourceList<T> page = new PagedResourceList<T>(....).withFilterValue("a","b");
-     */
-    public ForwardCursorPagedResourceList<T> withFilter(String key, String value) {
-        if (isNotBlank(key) && isNotBlank(value)) {
-            filters.put(key, value);
-        }
-        return this;
+    
+    @Deprecated
+    @JsonSerialize(using = DateTimeSerializer.class)
+    public DateTime getStartTime() {
+        return getDateTime(START_TIME);
     }
-    /**
-     * A convenience method for adding a date filter.
-     */
-    public ForwardCursorPagedResourceList<T> withFilter(String key, DateTime value) {
-        if (isNotBlank(key) && value != null) {
-            filters.put(key, value.toString());
-        }
-        return this;
+    @Deprecated
+    @JsonSerialize(using = DateTimeSerializer.class)
+    public DateTime getEndTime() {
+        return getDateTime(END_TIME);
     }
-    public List<T> getItems() {
-        return items;
+    @Deprecated
+    @JsonSerialize(using = DateTimeSerializer.class)
+    public DateTime getScheduledOnStart() {
+        return getDateTime(SCHEDULED_ON_START);
     }
-    public String getOffsetKey() {
-        return offsetKey;
+    @Deprecated
+    @JsonSerialize(using = DateTimeSerializer.class)
+    public DateTime getScheduledOnEnd() {
+        return getDateTime(SCHEDULED_ON_END);
     }
-    @JsonProperty("hasNext")
-    public boolean hasNext() {
-        return (offsetKey != null);
+    @Deprecated
+    public String getOffsetKey() {
+        return nextPageOffsetKey;
     }
```

Comment:
It should return the next page offset key, in every construction of the FCPRL, it is the offset for the next page. Of all the properties though, this is one is definitely being used by clients and they will need to be migrated to nextPageOffsetKey.